src/components/Search/Search.module.css

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2023 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.search {
+  border: 1px solid var(--cpd-color-border-interactive-secondary);
+  border-radius: 9999px;
+  height: 36px;
+  box-sizing: border-box;
+  color: var(--cpd-color-text-primary);
+  display: inline-flex;
+  gap: var(--cpd-space-2x);
+  align-items: center;
+  padding: var(--cpd-space-1-5x) var(--cpd-space-3x);
+  overflow: hidden;
```

Comment:
This shouldn't be necessary because there is an input field that holds the variable text.